CONTRIBUTING.md

```diff
@@ -36,9 +36,9 @@ To add a new rule, we suggest following these steps:
     - The test file with accepted phrases should be "(RuleName).good.txt"
     - The test file with denied phrases should be "(RuleName).bad.txt"
 8. If useful, create a folder named "original_files" and add relevant original files to that directory.
-9. Add the rule to the vale/styles/jargonLint or vale/styles/optional directory.
+9. Add the rule to the vale/styles/jargonLint or vale/styles/jargonExtra directory.
```

Comment:
I think this updates line should be enough to point that they are both directories